### PR TITLE
Address #1857: Fix typo/phrasing through AudioWorkletNode

### PR DESCRIPTION
**Pull request by rtoy**
* Fix typos and phrasing
 * Add/adjust some links

***
<a href="https://pr-preview.s3.amazonaws.com/rtoy/web-audio-api/pull/1902.html" title="Last updated on May 23, 2019, 7:30 PM UTC (6ccbdc6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/1902/09829e6...rtoy:6ccbdc6.html" title="Last updated on May 23, 2019, 7:30 PM UTC (6ccbdc6)">Diff</a>

### CODE REVIEW

**Comment by rtoy** [comment]
There are a few more items that I need to ask about before finishing this PR.

File: index.bs [hunk 1]
```diff
@@ -10035,16 +10036,16 @@ Constructors</h5>
 	: <dfn>port</dfn>
 	::
 		Every {{AudioWorkletProcessor}} has an associated
-		<code>port</code> which is a {{MessagePort}}. It is connected to the port on the
+		<code>port</code> which is a {{MessagePort}}. It is connected to a port on the
 		corresponding {{AudioWorkletProcessor}} object allowing
```
Comment (rtoy) [inline index.bs]: This is the port for the AudioWorkletProcessor.  Should this say it's connected to the port for associated AudioWorkletNode?

File: index.bs [hunk 2]
```diff
@@ -10246,27 +10247,27 @@ thread and the rendering thread.
...
 	5. Let <var>nodePort</var> be the value of
-		<var>messageChannel</var>'s <code>port1</code> attribute.
+		<var>messageChannel</var>'s {{MessageChannel/port1}} attribute.
 
 	6. Let <var>processorPortOnThisSide</var> be the value of
-		<var>messageChannel</var>'s <code>port2</code> attribute.
+		<var>messageChannel</var>'s {{MessageChannel/port2}} attribute.
 
-	7. Let <var>processorPortSerialization</var> be
+	7. Let <var>processorPortSerialization</var> be the result of
```
Comment (rtoy) [inline index.bs]: Is this right?

**Comment by rtoy** [comment]
I think I filed new issues for the items I was not sure about.  This is ready for review.

File: index.bs [hunk 3]
```diff
@@ -9560,8 +9560,8 @@ within a special scope named {{AudioWorkletGlobalScope}}.
 
 Each {{BaseAudioContext}} possesses exactly one
 {{AudioWorklet}}. Scripts imported using this {{AudioWorklet}}
-are run in one or more {{AudioWorkletGlobalScope}} global scopes,
-which are created to process audio associated with that
+are run in one {{AudioWorkletGlobalScope}} global scope,
```
Comment (hoch) [inline index.bs]: I think we can remove "global scope" at the end.

File: index.bs (hunk 1 above)
Comment (hoch) [reply index.bs]: This port is created by MessageChannel. So the association between port1 and port2 are already implied.

File: index.bs [hunk 4]
```diff
@@ -10067,7 +10068,7 @@ in the node.
 		* {{[[callable process]]}} is `true`.
 
 		* The {{AudioWorkletNode}} is [=actively processing=], meaning
-			ANY of the following states are met:
+			ANY of the following conditions are met:
```
Comment (hoch) [inline index.bs]: FYI: https://github.com/WebAudio/web-audio-api/pull/1875/files

File: index.bs (hunk 2 above)
Comment (hoch) [reply index.bs]: I believe so.

File: index.bs (hunk 4 above)
Comment (rtoy) [reply index.bs]: Yeah, well, I was only reading the current spec, not considering things PRs that might change things. :-)

File: index.bs (hunk 1 above)
Comment (rtoy) [reply index.bs]: Should I undo this change?

File: index.bs (hunk 3 above)
Comment (rtoy) [reply index.bs]: Done.

File: index.bs [hunk 5]
```diff
@@ -9900,33 +9901,33 @@ Dictionary {{AudioWorkletNodeOptions}} Members</h6>
...
 		{{NotSupportedError}} exception MUST be thrown if a
-		channel count is not in the valid range of AudioNode's
+		channel count is not in the valid range of an {{AudioNode}}'s
 		{{AudioNode/channelCount}}.
 
 	: <dfn>parameterData</dfn>
 	::
 		This is a list of user-defined key-value pairs that are used
-		to initialize {{AudioParam}} values in
+		to initialize {{AudioParam}} objects in
```
[the remark this answers is not in the document]
Comment (hoch) [reply index.bs]: LGTM

File: index.bs (hunk 1 above)
Comment (hoch) [reply index.bs]: `the port` is more specific. So yes, I think it's fine as-is.

File: index.bs [hunk 6]
```diff
@@ -9814,7 +9814,8 @@ This interface has "entries", "forEach", "get", "has", "keys",
 <pre class="idl">
 [Exposed=Window,
  SecureContext,
- Constructor (BaseAudioContext context, DOMString name, optional AudioWorkletNodeOptions options)]
+ Constructor (BaseAudioContext context, DOMString name,
+              optional AudioWorkletNodeOptions options)]
```
[the remark this answers is not in the document]
Comment (hoch) [reply index.bs]: LGTM

File: index.bs (hunk 5 above)
[the remark this answers is not in the document]
Comment (hoch) [reply index.bs]: LGTM

File: index.bs (hunk 1 above)
Comment (hoch) [reply index.bs]: I prefer "the port" because it's more specific.

File: index.bs (hunk 4 above)
Comment (hoch) [reply index.bs]: We can merge this first, and then I can handle the conflict in #1875.

File: index.bs (hunk 1 above)
Comment (rtoy) [reply index.bs]: Done.

Comment (rtoy) [reply index.bs]: done

**Pull request merged**